lib/workers/repository/dependency-dashboard.ts

```diff
@@ -15,25 +15,32 @@ interface DependencyDashboard {
   dependencyDashboardRebaseAllOpen: boolean;
 }
 
-function parseDashboardIssue(issueBody: string): DependencyDashboard {
+function checkRebaseAll(issueBody: string): boolean {
+  return issueBody.includes(' - [x] <!-- rebase-all-open-prs -->');
+}
+
+function getCheckedBranches(issueBody: string): RegExpMatchArray | null {
   const checkMatch = ' - \\[x\\] <!-- ([a-zA-Z]+)-branch=([^\\s]+) -->';
```

Comment:
instead of duplicate pass the regex via parameter? otherwise i don't see any benefit of this function

Reply:
@viceice I added a comment, this regex will be temporary until merging this feature (#15912).

Reply:
this regex will not be changed, the regex in `parseDashboardIssue()` line 32 will be updated next PR

Reply:
I would prefer to also move the `checked` loop inside this function too. and use one loop with `matchAll`